runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -541,18 +570,14 @@ private void hop(boolean previous)
 			 */
 			if (previous)
 			{
-				worldIdx--;
-
-				if (worldIdx < 0)
+				if (--worldIdx < 0)
```

Comment:
Please don't change unrelated code, especially if it only makes it harder to read.